File: OOP_Lab/Lab_0/lab_0_1.py
def calculateAverage(list):
    return sum(list) / len(list)


if __name__ == '__main__':
    num_list = []
    n = int(input("Enter number of elements: "))
    for n in range(n):
        num_list.append(float(input(f"Enter element no.{n+1}: ")))

    print(calculateAverage(num_list))

File: OOP_Lab/Lab_0/lab_0_1.cpp
#include<iostream>

using namespace std;

float calculateAverage(int arr[], int size) {
    int sum = 0;
    for (int i = 0; i < size; i++) {
        sum += arr[i];
    }
    return (float) sum / size;
}

int main() {
    int size = 0;
    cout << "Enter array size: ";
    cin >> size;

    int arr[size];
    for (int i = 0; i < size; i++) {
        cout << "Enter the element no." << i + 1 << ": ";
        cin >> arr[i];
    }

    float average = calculateAverage(arr, size);
    cout << "Average = " << average;
}